fast/stages/00-bootstrap/outputs-files.tf

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# tfdoc:file:description Output files persistence to local filesystem.
+
+resource "local_file" "providers" {
+  for_each        = var.outputs_location == null ? {} : local.providers
+  file_permission = "0644"
+  filename        = "${pathexpand(var.outputs_location)}/providers/${each.key}-providers.tf"
+  content         = each.value
+}
+
+resource "local_file" "tfvars" {
+  for_each        = var.outputs_location == null ? {} : { 1 = 1 }
+  file_permission = "0644"
+  filename        = "${pathexpand(var.outputs_location)}/tfvars/00-bootstrap.auto.tfvars.json"
+  content         = jsonencode(local.tfvars)
+}
+
+resource "local_file" "tfvars_globals" {
+  for_each        = var.outputs_location == null ? {} : { 1 = 1 }
+  file_permission = "0644"
+  filename        = "${pathexpand(var.outputs_location)}/tfvars/globals.auto.tfvars.json"
+  content         = jsonencode(local.tfvars_globals)
+}
+
+resource "local_file" "workflows" {
```

Comment:
This is Github specific. For Gitlab it would be .gitlab-ci.yml in the root of the repo. We could add this in the workflows folder - but I just wanted to mention it. gitflow-workflow.yaml would have to be renamed to .gitlab-ci.yml.

Reply:
Ok, let me find the least disruptive way of doing this.

Reply:
Once you add the Gitlab workflow I'll fix via locals.

Reply:
this is now mentioned as part of the CI/CD steps for configuring repositories in the stage 0 README